cfme/configure/configuration/diagnostics_settings.py

```diff
@@ -343,11 +349,20 @@ class ServerCollectLogsView(ServerDiagnosticsView):
 
     @property
     def is_displayed(self):
+        tree = self.accordions.diagnostics.tree.currently_selected
+        selected_server = tree[-1].split(': ')[-1]
+        try:
+            name, sid, current = selected_server.split()
```

Comment:
**Required** I don't think this is a good assumption to make, what is name or sid ended up raising a ValueError. Perhaps look at the length before the split.